content/v2/registrar/whois-privacy.markdown

```diff
@@ -17,6 +17,12 @@ Enable and disable WHOIS privacy on registered domains.
 
 Get the WHOIS privacy details for a domain.
 
+<warning>
+**The deprecation period for this endpoint began on 01 May 2023.**
+
+Now you can enable WHOIS Privacy protection for any of your domains any time, as long as the domain is registered with us and the TLD (Top Level Domain) supports the WHOIS Privacy service, the WHOIS Privacy service will stay enabled until it is disabled.
+</warning>
```

Comment:
WDYT about pointing ppl to https://developer.dnsimple.com/v2/domains/#domain-attributes as that's where they would need to look if they want to understand if whois privacy is enabled

Reply:
@juankuquintana I would run this new content through @itsalyse. IMO it would be more valuable to announce the end date instead of the begin date of the deprecation.

I agree with @OleMchls suggestion about pointing to the end point that shows if the privacy is supported for their TLD.

Reply:
I think including both dates is a good idea - having the end date would definitely be helpful here